Sources/Subs.php

```diff
@@ -899,6 +899,28 @@ function permute($array)
 	return $orders;
 }
 
+// Rearranges all parameters to be in the right order.  Returns TRUE if no parameters are leftover.
+function_param_order($message, &$parameters, &$out)
```

Comment:
Whoops!  I meant to use the following:
function fix_param_order($message, &$parameters, &$out)